config/config.js

```diff
@@ -81,6 +81,12 @@ var initGlobalConfigFolders = function(config, assets) {
     config.folders.client = getGlobbedPaths(path.join(process.cwd(), 'modules/*/client/'), process.cwd().replace(new RegExp(/\\/g),'/'));
 };
 
+var transformLibPaths = function(paths) {
```

Comment:
why not just `function transformLibPaths` instead of putting it in a variable? Having it in a variable makes using an IDE harder and less useful in addition to just being unnecessary, no?